### ISSUE TITLE
Support for Nexus Collections

### ISSUE THREAD

**Issue by fluteds** [Floogen/Stardrop#100]
I know this mod is still in Beta but I'd love to have some support for downloading/managing Nexus Collections in the future if possible. Thank for for making this mod it's a life saver.

**Comment by Floogen** [comment]
I don't believe Nexus Mods has opened up their API for collections, so Stardrop does not have a viable avenue to support them.

If anyone has information for pulling collections via Nexus Mods API, I will gladly investigate further.

**Comment by srsobieraj** [comment]
I know it's been years, but Collections are just a list of mods that are grouped together.  Stardrop could scrape the Collection URL for the names/url of the mods, then create a folder and manifest/index file for them, and in the UI indicate how many/which mods were missing, similar to how Missing Requirements works right now.

This is effectively what happens in the new Nexus Mod Manager with the free subscription anyways - it just queues each mod in the collection for download, but you have to click on the Download button manually for each one.

Right now the workaround is to just download each component mod and put them in a folder manually, which isn't very onerous, but Stardrop does throw a semi-cryptic error when you try to open a NXM link for a collection.

**Comment by hotcereal** [comment]
With the sudden closure of the Nexus Mod Manager and Nexus providing no path for people to switch to Vortex, coupled with Vortex being a horribly built app, I hope the above can be implemented into a future version of Stardrop. 

I managed to use NMM originally to add the collection, then moved over to Stardrop, so I’m relatively set myself, but I imagine newcomers will want a similar level of ease in downloading an entire collection too.
